modules/axruntime/src/mp.rs

```diff
@@ -11,7 +12,7 @@ pub fn start_secondary_cpus(primary_cpu_id: usize) {
     let mut logic_cpu_id = 0;
     for i in 0..SMP {
         if i != primary_cpu_id {
-            let stack_top = virt_to_phys(va!(unsafe {
+            let stack_top = virt_to_phys(VirtAddr::from(unsafe {
```

Comment:
you just revert it back (dog head manually

Reply:
🙃

Reply:
I removed `va` and `pa` exports in `axhal::mem`, so we need to import the `memory_addr` dependency if use `va!`.

Do you think it is OK?

Reply:
I agree with the former